index.js

```diff
@@ -65,8 +65,8 @@ class bluetoothLE extends EventEmitter {
       this.server = await this.device.gatt.connect();
       console.log('Connected.');
 
-      this.glucoseService = await this.server.getPrimaryService('glucose');
       this.deviceInfoService = await this.server.getPrimaryService('device_information');
+      this.glucoseService = await this.server.getPrimaryService('glucose');
```

Comment:
I'm curious about this change. On the face of it, looks like no-op. Was there a particular reason?

Reply:
No particular reason, IIRC some services were failing because I didn't have the right permissions in place, so I was swapping their order to see if anything changed. Turns out I just needed to get the permissions working properly first.